defaults/main.yml

```diff
@@ -93,3 +92,97 @@ paxctld_configs_dist:
+  camFEyEY310dGb2SvcVpWMX9wB0NH/BDlR+7vSjIPRLSt9PM8oE9NoviehbZ5h6T
+  a/OJcsY84JmcMAsji770/uy9XXPqtMno8DJb/Os/Ps8H4DF4HsQEPxlqD5WX99uw
+  4p7m0NTpJD9E/hSZfslIZNxkf1gQ2lAonM3HZ++WPEL1L96UZ6pqFl0aXU7BSYiV
+  330/MnRLx8dKoR51nMcnwQVKN8JfXQlZEb2ChuIoiYxpW/CAx1vgcQsFh50lHDAS
+  A+6p19fRmAPenh1SgJZvYctF7XbG5pyiSJqQVbAU9khzpnMZ2it6k4Mg6hG6HCi1
+  EORb3ipAM6hwROe4lre/NWA3qEhBJFnMMInWXG1gbViNIze/MZOlpPjKDBstqver
+  MUy2lpKwE/AEJCWs6SoSRIlIF7vIwnLiWbBY8OsY8eSN8LRwxidClpsO5SFwJYBp
+  x/L4NhGiDmxy1MQt3C/84S0kpFpdu+Ex4smW3gw7yCYkl+P6F2EeSkIQBN92vdAf
+  HGIAjZ65a3G0DJ7YJyp0YrFKkyI3Pr3iSn3JisCqtLQeUzbnqAZC7h9GYB6JAjcE
+  EwEKACEFAlPSs9YCGwMFCwkIBwMFFQoJCAsFFgIDAQACHgECF4AACgkQRNHA+CUl
+  /kmeEQ/+NyNu1o4nZ+0E02LAlSU5Ev5BzEMQ+w/hgGIj1d3u0CB1ShcHJnwrAugM
+  ysSK1E335ea1I3a/VmqAsyYQp0nASGqmS4yjK6Q3Z1bmwxKESCAO7q7UsRrZdcLL
+  iSGJ4CNze+w5Y83jstFc+lN5gIPDlyCz/5v8ySRDmgrdXQZFoNPJNWmGEGQaUp2X
+  oV5gOkNCgbt9nsWhSXKWygbbMIjX88OxzIxDTxym2G3WecprN3FLXpW1OQ+8l0/Z
+  WcypjVtCGvSQO/b8SCupoCzzZpESBX2kedJoK6Tv0pXgc1r/dPVZoZdfg1qhXvWp
+  0P/tZIMomwvRq7GxoNax8GSfuePErO1SVJr2YuUf+JmYgQuaozQMUrUQoOOYVJXd
+  RBCNUx7VV3tsp3WzE6Btc53o4Y74WUD1WOvkjkAIUaUXO8ycH4l/tQ39qymxBkZb
+  FNa3PUjpb1gG8G/9QUrBCMPum+3WLakimpm83Haj813+ILxy4IcMJPx6kzee83yW
+  2TUZAhcX1ztM4qwkY84bxqbJ9l42bBWXglJirp+L7Vjgn2CNeHZhzuc/55e91JM5
+  OuFhONgzDrqtzwG2Xu6MOZ76vGJj+ymqagXb+chu/3zzoPQAVldd6BbdoombWu25
+  17mrO3HPKbwUkc2H2xbCbGhGK1EKtzBObMDitD6t+CWpntKf+8S5Ag0EUn+wdAEQ
+  AKD+s7FwKTBjchxDpTmwQsC7A+3gPiqJfmgUaW3gSe6pejiLuffLrrTJTOU1lwVz
+  z58S010bVzLmEpi8I3mK3KRnf/GzshObRnoVFDeGMdiOfJWWV/vLHTeW3576GGq4
+  ZQ070KwUX8XoctoTUXWi2PSl69RGFOoZGafKY+GfdnI0ui1tt9LMRArsrozs9YsD
+  5HWxT0+pj0f7BfPjrmbjlRfpGwLkGW3hRb3O/Tq8T87HyBvl4VYXVWWoamI7JWmk
+  8Z0i+VnikQF7Tn8BW2XUkwycl/OCDBGMSZrtnMG1VOghRvDn4QK/ul6uESpL09it
+  Lm6PvhJLaIAf7CdDmuiqWrabcHSLt7qt7gybYAmHUcyfG76VsJfqovT1PG7WtK17
+  OION+YgvnlC8v0dk2Zh1tneXuctvy5ggiv4ZHc8iSKhe90TF4pPxnRNUpup+WVjc
+  ZvRWCr4YmsjWN2dMgUzYHjJp44VDEQwzt5uuuS4q7HR5cH0hCcUbbq3G5d4sbC3E
+  NP2YUTqyxNE8PQE8926F4G4d4leIsV6OZGEvPZLKcNcKHckmHVU6YrXuEq65eEHi
+  Eg0SGnIxY6de+r6XMmvPAA1KQkBW+5krjyAB7PGLsUqVG0J9n85/XuXGb8BGIsMa
+  DjNccliVIZLV+w2rzupMhm/+V6E6jeNePDiWoG8vYg6/ABEBAAGJAh8EGAEKAAkF
+  AlJ/sHQCGwwACgkQRNHA+CUl/kkGIw//e6oDj0HzvBa+ctoiY7nXlglyqeOYjaK+
+  jf/gwsHMzzFJlZ/8Ml+xMN2jgID/LeJ2deRbQAeFUMpuJVoHTSRweXirflbS/M0p
+  sroLYoJbgzH4jY5ZZwD4pFP0KaF2rVrnihud7gbxvlOfzdxe06tW5Xe6X0y3C8yq
+  42+EmJg3tSYl64Sjagg6YIgEiyRimKFLz/b/bZTsa6p/87KCLf1IrMtUcyWx5Y8k
+  zRC0TOaH+hukGB7n9pbNqoLpNV+qCw1Lpdiqr8IEsV5kuAmNxa43eLzclIX2697T
+  mb9B8T0MHAqLpyFrGm/M6uh9AvAZp7EWaQAcIOL1KPj8KNm5ZiV5DQU4eDNcXqyK
+  QtTVLhWZ15Oy5zlwo6I1eKvCk4LcPfw9C+h+I2/xaMmNGEU0GOWPrGeo6K0wPTr6
+  9MfX0KSolm2XTCs8mNJbRYt9X2WjiM/38YInkHg09o5rUtAun7Tn/oRZOpTWNFrf
+  7AykszcJY+lN5GUAlJoml4rAze9FqnKVSNIuPntXjodsY94sEvycOfIUMamPm1PG
+  DhJyOvJMyAbC5CSigp00rgzTSV88WFsDv/uBVUvFghNvObYYdAess6zCvrP4ge5e
+  rDkzZpiYUYN6J/3gAGNw+Hj5ZqLlNmprLZBEsrK8FXhYb9+yh2YJNy2hD5AyGjMt
+  1bFx6/2R4TU=
+  =nWod
+  -----END PGP PUBLIC KEY BLOCK-----
```

Comment:
Vars are great, I'd've stuck this in a file that we can clobber, then it's easier to import ad-hoc as well.